charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.34.3
+
+* If set allow arbitrary podlabels on pods.
```

Comment:
```suggestion
* If set, allow arbitrary podlabels on pods.
```
Something feels missing. If what is set? If a config option is set (in which case say "If X option is set, allow ...")? Or if arbitrary pod labels are set (in which case say "If arbitrary podlabels are set, allow them on pods" or "Allow optional arbitrary podlabels on pods")?

Reply:
I've made the requested changes to the changelog.